capirca/lib/policy.py

```diff
@@ -2794,8 +2825,10 @@ def p_interface_spec(p):
   """
   if p[1].find('source-interface') >= 0:
     p[0] = VarType(VarType.SINTERFACE, p[4])
-  elif p[1].find('destination-interface') >= 0:
+  if p[1].find('destination-interface') >= 0:
     p[0] = VarType(VarType.DINTERFACE, p[4])
+  elif p[1].find('interface') >= 0:
```

Comment:
Now that we're not using this field at all, could you please clean up all references to "interface"?